src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -212,7 +213,31 @@ public async Task<int> CreateBuildAsync(string repositoryUrl, string branch, str
                 Dependencies = dependencies,
             });
 
-            return build.Id;
+            return build;
+        }
+
+        public async Task<string> GetDarcBuildAsync(int buildId)
+        {
+            string buildString = await RunDarcAsync("get-build", "--id", buildId.ToString());
+            return buildString;
+        }
+
+        public async Task<string> UpdateBuildAsync(int buildId, string updateParams)
+        {
+            string buildString = await RunDarcAsync("update-build", "--id", buildId.ToString(), updateParams);
+            return buildString;
+        }
+
+        public async Task<string> GatherDrop(int buildId, string outputDir, bool includeReleased)
+        {
+            if (includeReleased)
+            {
+               return await RunDarcAsync("gather-drop", "--id", buildId.ToString(), "--dry-run", "--include-released", "--output-dir", outputDir);
+            }
+            else
+            {
+                return await RunDarcAsync("gather-drop", "--id", buildId.ToString(), "--dry-run", "--output-dir", outputDir);
+            }
```

Comment:
This could be something like:
```suggestion
            var args = new []{"gather-drop", "--id", buildId.ToString(), "--dry-run", "--output-dir", outputDir};
            if (includeReleased)
            {
                args = args.Append("--include-released").ToArray();
            }
            return await RunDarcAsync(args);
```